eng/Packages.Data.props

```diff
@@ -196,4 +196,8 @@
     <PackageReference Update="System.Reflection.Emit" Version="4.7.0" />
   </ItemGroup>
 
+  <!-- InheritDoc-->
+  <ItemGroup Condition="'$(InheritDocEnabled)' == 'true'">
```

Comment:
The condition isn't necessary. If the package wasn't included the `Update` does nothing. I wouldn't duplicate conditions more than necessary as it gets harder to maintain.